pyte/screens.py

```diff
@@ -1240,13 +1241,16 @@ def prev_page(self):
             mid = min(len(self.history.top),
                       int(math.ceil(self.lines * self.history.ratio)))
 
-            self.history.bottom.extendleft(reversed(self.buffer[-mid:]))
+            extension = [self.buffer[line] for line in reversed(range(self.lines - mid, self.lines))]
```

Comment:
Consider inlining, there's no point of having this as a temp variable named `extension`.